ledger/src/ui/test/bootloader/mock/mock.h

```diff
@@ -0,0 +1,91 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 RSK Labs Ltd
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ */
+
+#ifndef _MOCK_H
+#define _MOCK_H
+
+#include <assert.h>
+#include <stdint.h>
+#include "os_exceptions.h"
+
+#define CHANNEL_APDU 0
+#define INS_ATTESTATION 0x50
+#define INS_SIGNER_AUTHORIZATION 0x51
```

Comment:
We should probably grab these from the actual values if we can.

Reply:
These values are defined in` attestation.h` / `signer_authorization.h`, adding these files would introduce a lot of dependencies (like the build flag mentioned on the comment above, private and public key type definitions, and we would need the implementation for all public functions in these modules). It is possible to add this, but I feel like this would make this test even more bloated than it is now.

If we ever change these defines, the tests will fail and we just need to update the definitions accordingly. What do you think?